File: py101-basics/exercises/easy_3/mad_libs.py
# Given a noun, verb, adverb, and adjective
# print multi-line f-string taking user input, making story
noun = input("Enter a noun: ")
verb = input("Enter a verb: ")
adj = input("Enter an adj.: ")
adv = input("Enter an adv.: ")

print(f"""
Do you {verb} your {adj} {noun} {adv}? That's hilarious!
The {adj} {noun} {verb}s {adv} over the lazy dog.
The {noun} {adv} {verb}s up to Joe's {adj} turtle.
      """)

File: js210/exercises/easy_2/madlibs.js
const rlSync = require('readline-sync');
const noun = rlSync.question('Enter a noun: ');
const verb = rlSync.question('Enter a verb: ');
const adjective = rlSync.question('Enter an adjective: ');
const adverb = rlSync.question('Enter an adverb: ');

const sentence1 = `Do you ${verb} your ${adjective} ${noun} ${adverb}? That's hilarious!`;
const sentence2 = `The ${adjective} ${noun} ${verb}s ${adverb} over the lazy dog.`;
const sentence3 = `The ${noun} ${adverb} ${verb}s up ${adjective} Joe's turtle.`;

console.log(sentence1);
console.log(sentence2);
console.log(sentence3);